pandas/io/tests/parser/python_parser_only.py

```diff
@@ -171,3 +171,17 @@ def test_read_table_buglet_4x_multiindex(self):
             columns=list('abcABC'), index=list('abc'))
         actual = self.read_table(StringIO(data), sep='\s+')
         tm.assert_frame_equal(actual, expected)
+
+    def test_temporary_file(self):
+        # GH13398
+        data1 = "0 0"
+
+        from tempfile import TemporaryFile
+        new_file = TemporaryFile("w+")
+        new_file.write(data1)
+        new_file.flush()
+        new_file.seek(0)
+
+        result = self.read_csv(new_file, sep=r"\s+", header=None)
```

Comment:
Hmm...how odd.  It's because we do special handling of `\s+`, and you might be seeing it (while I am not) due to LOCALE.  To be safe, I would recommend doing `sep="\s*"`.